samples/subsys/ipc/ipm_imx/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _ipm_imx:
+
+i.MX IPM example
+################
+
+Overview
+********
+A simple example that can be used with multicore i.MX SoC's containing the
+Messaging Unit peripheral. It reads the received data from the Messaging Unit
+using the IPM and transmits it back unchanged. The information about
+the received data is printed to the console. When sending the data back,
+it blocks until the data are read by the remote side.
+
+The i.MX Messaging Unit peripheral consists of 4 32-bit transmit and receive
+registers. The sample uses the option CONFIG_IPM_IMX_MAX_DATA_SIZE_4, which
```

Comment:
You can automatically create a link to the config option documentation by writing this as:

    ... uses the option :option:`CONFIG_IPM_IMX_MAX_DATA_SIZE_4`, which